apps/academy-wep/components/learn/learning-module-card.tsx

```
@@ -0,0 +1,177 @@
+          <div className="flex items-center justify-between text-sm text-gray-500 mb-4">
+            <div className="flex items-center">
+              <BookOpen className="h-4 w-4 mr-1" />
+              {module.completed}/{module.lessons} lessons
+            </div>
+            <div className="px-2 py-0.5 bg-gray-100 rounded-full text-xs">{module.level}</div>
+          </div>
+
+          <div className="flex flex-wrap items-center gap-2 mb-4 min-h-[28px]">
+            {module.tags.map((tag, index) => (
+              <span
+                key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)